### ISSUE TITLE
Getting strange error regarding subchannels

### ISSUE THREAD

**Issue by schmidt-sebastian** [grpc/grpc-node#1250]
Manual copy of: https://github.com/googleapis/nodejs-firestore/issues/890

#### Environment details

  - OS: Mac OS 10.15 Beta (19A573a)
  - Node.js version: v12.10.0
  - npm version: 6.10.3
  - `@google-cloud/firestore` version: 3.3.4

#### Steps to reproduce

I am unsure how to reproduce as there doesn't seem to be much indication in my logs.
The problem I have is that I keep getting this error popping up:
```
(node:33) UnhandledPromiseRejectionWarning: Error: 14 UNAVAILABLE: Failed to start call on picked subchannel
    at Object.callErrorFromStatus (/api/node_modules/@grpc/grpc-js/src/call.ts:79:24)
    at Http2CallStream.<anonymous> (/api/node_modules/@grpc/grpc-js/src/client.ts:155:18)
    at Http2CallStream.emit (events.js:333:22)
    at Http2CallStream.EventEmitter.emit (domain.js:485:12)
    at /api/node_modules/@grpc/grpc-js/src/call-stream.ts:183:14
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

**Comment by schmidt-sebastian** [comment]
CC @jakeleventhal

**Comment by murgatroid99** [comment]
I have published grpc-js version 0.6.16 with a fix for this, but in some cases you will just see a different error with the same error code.
